purchase_order_approved/README.rst

```diff
@@ -0,0 +1,71 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=======================
+Purchase Order Approved
+=======================
+
+This module extends the functionality of purchases adding a new state
+*Approved* in purchase orders. Additionally add the possibility to set back to
+draft a purchase order in all the states previous to *Purchase Order*.
+
+In this new *Approved* state:
+
+* You cannot modify the purchase order.
```

Comment:
I think what you propose is redundant, since I started a bullet list with the sentence "In this new state:" all the bullets are supposed to continue this starting phrase. In this case: " In this new state you cannot modify the purchase order."